Dockerfile

```diff
@@ -1,21 +1,14 @@
 FROM node:10-alpine as builder
 
 WORKDIR /app
-RUN apk add yarn
 
 COPY . ./
 
-# environment variable $KILT_NPM_AUTH_TOKEN must be provided when building the image:
-# docker built --build-arg KILT_NPM_AUTH_TOKEN=xxx ...
-ARG KILT_NPM_AUTH_TOKEN=""
```

Comment:
Why can these go away now?

Reply:
Before going public with our repos, our sdk was a private npm package. The only way to access private npm packages is by defining an npm auth token. Since we are working with a public package now, this is not necessary anymore